<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskStashW</class>
 <widget class="QMainWindow" name="TaskStashW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">cchelper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="F1Button">
        <property name="text">
         <string>?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="settingButton">
        <property name="text">
         <string>S</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="findButton">
        <property name="text">
         <string>F</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="newButton">
        <property name="text">
         <string>N</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="delButton">
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <widget class="QToolButton" name="buildButton">
        <property name="text">
         <string>4</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="runButton">
        <property name="text">
         <string>5</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="stopButton">
        <property name="text">
         <string>T</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="submitButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>6</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="arxivButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>7</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="logButton">
        <property name="text">
         <string>Log</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="TCButton">
        <property name="text">
         <string>Test Cases</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="VButton">
        <property name="text">
         <string>Verdicts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="SButton">
        <property name="text">
         <string>Solver</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="GButton">
        <property name="text">
         <string>Generator</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="JButton">
        <property name="text">
         <string>Jurger</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="CodeEditor" name="SEditor"/>
      <widget class="CodeEditor" name="GEditor"/>
      <widget class="CodeEditor" name="JEditor"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QWidget</extends>
   <header location="global">cchelper.fileeditor.codeeditor</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
